impeller/golden_tests/metal_screenshot.mm

```diff
@@ -17,7 +17,7 @@
   CGImageRelease(cg_image_);
 }
 
-const UInt8* MetalScreenshot::GetBytes() const {
+const uint8_t* MetalScreenshot::GetBytes() const {
```

Comment:
(non-blocking) for my own curiosity, what is the difference between UInt8 and uint8_t here?

Reply:
UInt8 comes from Apple's c headers and used in Apple code, `uint8_t` comes from `stdint.h`.